store-implementation/accumulo-store/src/test/java/uk/gov/gchq/gaffer/accumulostore/utils/AddUpdateTableIteratorTest.java

```diff
@@ -130,18 +124,25 @@ public void shouldReturnStoreClassNameNotFoundWhenStorePropsIsEmpty() throws Exc
 
         // When / Then
         assertThatIllegalArgumentException()
-                .isThrownBy(() -> AddUpdateTableIterator.main(args))
-                        .withMessage("The Store class name was not found in the store properties for key: gaffer.store.class");
+            .isThrownBy(() -> AddUpdateTableIterator.main(args))
+            .withMessage("The Store class name was not found in the store properties for key: gaffer.store.class");
     }
 
     @Test
     public void shouldReturnInvalidFilePathErrorWhenPathDoesNotExist() throws Exception {
-        // Given
+        String expectedMessage;
+
         final String[] args = {GRAPH_ID, SCHEMA_DIR, "invalid/file/path", "update", FILE_GRAPH_LIBRARY_TEST_PATH};
 
-        // When / Then
+        // Without the following the return message will have / instead of \ when run on Windows";
+        if (IS_OS_WINDOWS) {
+            expectedMessage = "Failed to load store properties file : invalid\\file\\path";
+        } else {
+            expectedMessage = "Failed to load store properties file : invalid/file/path";
+        }
+
         assertThatExceptionOfType(RuntimeException.class)
-                .isThrownBy(() -> AddUpdateTableIterator.main(args))
-                .withMessage("Failed to load store properties file : invalid/file/path");
+            .isThrownBy(() -> AddUpdateTableIterator.main(args))
+            .withMessage(expectedMessage);
```

Comment:
Regex based matching can be used here to cover both cases concisely, without custom variables:
```java
 assertThatExceptionOfType(RuntimeException.class)
     .isThrownBy(() -> AddUpdateTableIterator.main(args))
     .withMessageMatching("Failed to load store properties file : invalid[\\\/]{1,2}file[\\\/]{1,2}path");
```

Reply:
Done, solution slightly different due to regex problems.